### PR TITLE
Jsgui tweaks

### PR DESCRIPTION
**Pull request by ahgittin**
* config+sensors tooltip renders nicely
* catalog items listed in accordion view don't wrap
* yaml preview window focuses at the top not the bottom (for long blueprints)

The second one needs tested on windows to ensure scrollbars don't clutter.

You can test 2 and 3 with:

```
brooklyn.catalog:
  items:
  - id: foo-bar-baz-foo-bar-baz-foo-bar-baz-foo-bar-baz-foo-bar-baz-foo-bar-baz-foo-bar-baz_boo
    itemType: template
    item:
      location: localhost
      services:
      - type:  brooklyn.entity.basic.BasicEntity
        brooklyn.config:
         x:
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
          blah
```

### CODE REVIEW

**Comment by aledsage** [comment]
@sjcorbett could you review this please?

@ahgittin asfbot failure is unrelated - fixed in #702, introduced in #700 

@ahgittin do you want this merged for 0.7.0 release, or can it be deferred?

**Comment by ahgittin** [comment]
Yes, I'd like this in 0.7.0, it fixes several glitches in the GUI.

And I have one more on catalog / user-start experience which will come later today which I'd love to get in.  Unfortunately that one isn't such a small one.  :(

File: usage/jsgui/src/main/webapp/assets/tpl/apps/config-name.html [hunk 1]
```diff
@@ -18,7 +18,17 @@
 under the License.
 -->
 
+<% if (description || type) { %>
 <span class="config-name" rel="tooltip" data-placement="left"
-    title="<% if (description) { %><b><%- description %></b><br/><% } %>(<%- type %>)">
+    title="<% 
+         if (description) { %><b><%- description %></b><br/><% } 
+         if (type) { %>(<% 
+           if (type.startsWith("java.lang.") || type.startsWith("util.")) type = type.substring(10);
```
Comment (sjcorbett) [inline usage/jsgui/src/main/webapp/assets/tpl/apps/config-name.html]: Why `substring(10)` if `type.startsWith("util.")`?

File: usage/jsgui/src/main/webapp/assets/tpl/apps/sensor-name.html [hunk 2]
```diff
@@ -18,7 +18,17 @@
 under the License.
 -->
 
+<% if (description || type) { %>
 <span class="sensor-name" rel="tooltip" data-placement="left"
-    title="<% if (description) { %><b><%- description %></b><br/><% } %>(<%- type %>)">
+    title="<% 
+         if (description) { %><b><%- description %></b><br/><% } 
+         if (type) { %>(<% 
+           if (type.startsWith("java.lang.") || type.startsWith("util.")) type = type.substring(10);
```
Comment (sjcorbett) [inline usage/jsgui/src/main/webapp/assets/tpl/apps/sensor-name.html]: As before.

**Comment by neykov** [comment]
Horizontal scroll looks fine on Windows:
windows catalog

There's a vertical scrollbar though, caused by a `scrollbar-y: scroll` rule.

File: usage/jsgui/src/main/webapp/assets/tpl/apps/config-name.html (hunk 1 above)
Comment (ahgittin) [inline usage/jsgui/src/main/webapp/assets/tpl/apps/config-name.html]: good spot.  should be `java.util.` of course!

**Comment by ahgittin** [comment]
@neykov thanks for the screen shot -- i'd rather the scrollbar wasn't there but i think worst case we can live with it; wdyt?

the only CSS rule intro'd here applies to the individual rows -- and on mac the vert scroll bar is per "accordion-item" (e.g. applications, or entities; but not all for) -- so i'm wondering if that is introduced here?

**Comment by ahgittin** [comment]
@sjcorbett fix pushed for the `util.` issue (in two places)

good to merge?

**Comment by sjcorbett** [comment]
Fire away.

File: usage/jsgui/src/main/webapp/assets/css/base.css [hunk 3]
```diff
@@ -1114,6 +1114,11 @@ tr.app-add-wizard-config-entry {
     margin-bottom: 6px;
     background-color: #F0F0F0;
     border-radius: 5px;
+    overflow-y: scroll;
```
Comment (neykov) [inline usage/jsgui/src/main/webapp/assets/css/base.css]: This causes the vertical scrollbar on Windows. It's on the element that contains all accordion elements. Looks like it's not really needed? It's not visible on OS X due to the transparent scrollbars.

Not important of course, can be merged as is.

Comment (ahgittin) [inline usage/jsgui/src/main/webapp/assets/css/base.css]: Why the hell did I add this?!?

Comment (ahgittin) [inline usage/jsgui/src/main/webapp/assets/css/base.css]: I'm gonna sneak a remove in to master directly.

**Pull request merged**